_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -0,0 +1,431 @@
+---
+id: fd3a94
+name: Links with identical accessible names and context serve equivalent purpose
+rule_type: atomic
+description: |
+  This rule checks that links with identical accessible names and context resolve to the same or equivalent resources.
+accessibility_requirements:
+  wcag20:2.4.4: # Link Purpose (In Context) (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Anne Thyme Nørregaard
```

Comment:
I think we should put Anne as previous author. She has been out of the loop 6 months.